src/lib/list/list.ts

```diff
@@ -175,3 +204,122 @@ export class MdListItem implements AfterContentInit {
     return this._element.nativeElement;
   }
 }
+
+@Component({
+  moduleId: module.id,
+  selector: 'md-list-option',
+  host: {
+    'role': 'option',
+    'class': 'mat-list-item',
+    '(focus)': '_handleFocus()',
+    '(blur)': '_handleBlur()',
+    '(click)': 'onchange()',
+    '(keydown)':'onKeydown($event)',
+    '[tabIndex]': 'disabled ? -1 : 0',
+    '[attr.aria-selected]': '',
```

Comment:
Missing binding expression for `aria-selected`. The expression needs to resolve to a _string_ of either `'true'` or `'false'` (see how existing `MdOption` does this).

You'll also need a binding for `aria-disabled` based on the disabled state